src/foreign_metrics.rs

```diff
@@ -166,4 +199,23 @@ impl Foreign {
 
         true
     }
+    pub fn record_floating_point_event(&mut self,
+                                       event_category: &str,
+                                       event_action: &str,
+                                       event_label: &str,
+                                       event_value: f64)
+                                       -> bool {
+        let ev: &Arc<Mutex<Events>> = match self.events {
+            Some(ref v) => v,
+            None => {
+                logger().log(LogLevelFilter::Error, "init_metrics has not been called");
+                return false;
+            }
+        };
+        let mut events_mut = ev.lock().unwrap();
+        events_mut.insert_floating_point_event(event_category, event_action, event_label, event_value);
+        logger().log(LogLevelFilter::Info, "Recorded event");
```

Comment:
Sorry, I meant to say instead of 
logger().log(LogLevelFilter::Info, "Recorded event");
that
logger().log(LogLevelFilter::Info, "Recorded floating point event");
would be better.

The function would still be called `record_floating_point_event`